client/state/timezones/actions.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+import {
+	TIMEZONES_RECEIVE,
+	TIMEZONES_REQUEST,
+	TIMEZONES_REQUEST_FAILURE,
+	TIMEZONES_REQUEST_SUCCESS
+} from 'state/action-types';
+
+export const timezonesRequestAction = () => ( {
+	type: TIMEZONES_REQUEST
+} );
+
+export const timezonesRequestSuccessAction = () => ( {
+	type: TIMEZONES_REQUEST_SUCCESS
+} );
+
+export const timezonesRequestFailureAction = error => ( {
+	type: TIMEZONES_REQUEST_FAILURE,
+	error
+} );
+
+/*
+ * The data given in as the function parameter has already been checked
+ * from the `timezones` data-layer.
+ * Take a look to requestTimezones function for more information.
+ */
+export const timezonesReceiveAction = ( data ) => ( {
```

Comment:
Do you mind suggest a name for this action please? I'll be more confident with your criteria that with mine.